src/views/catalog/wizard/tabs/environment/components/WizardEnvironmentTabFooter.tsx

```diff
@@ -0,0 +1,71 @@
+import * as React from 'react';
+
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import { ActionGroup, Alert, AlertActionCloseButton, Button } from '@patternfly/react-core';
+
+import './WizardEnvironmentTabFooter.scss';
+
+type WizardEnvironmentTabFooterProps = {
+  error?: any;
+  isSaveDisabled?: boolean;
+  onSave: () => void;
+  onReload: () => void;
+};
+
+const WizardEnvironmentTabFooter: React.FC<WizardEnvironmentTabFooterProps> = ({
+  error,
+  isSaveDisabled,
+  onSave,
+  onReload,
+}) => {
+  const { t } = useKubevirtTranslation();
+  const [success, setSuccess] = React.useState(false);
+  const [apiError, setApiError] = React.useState<any>();
+
+  const _onSave = async () => {
+    try {
+      await onSave();
+      setSuccess(true);
+      setApiError(undefined);
+    } catch (onSaveError) {
+      setApiError(onSaveError);
+    }
+  };
+
+  return (
+    <div className="wizard-environment-tab__buttons">
```

Comment:
please use `Stack hasGutter` for spacing between the alerts and the action group.
Also, please add a maxWidth to the Alerts